docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file.md

```diff
@@ -31,162 +31,207 @@ Steps
-                    "prefix":  "my_optional_data_docs_prefix",
+        "validations_S3_store": {
+            "class_name": "ValidationsStore",
+            "store_backend": {
+                "class_name": "TupleS3StoreBackend",
+                "bucket": "my_validations_store_bucket",
+                "prefix": "my_validations_store_prefix",
+            },
+        },
+        "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+    },
+    expectations_store_name="expectations_S3_store",
+    validations_store_name="validations_S3_store",
+    evaluation_parameter_store_name="evaluation_parameter_store",
+    data_docs_sites={
+        "s3_site": {
+            "class_name": "SiteBuilder",
+            "store_backend": {
+                "class_name": "TupleS3StoreBackend",
+                "bucket":  "my_data_docs_bucket",
+                "prefix":  "my_optional_data_docs_prefix",
+            },
+            "site_index_builder": {
+                "class_name": "DefaultSiteIndexBuilder",
+                "show_cta_footer": True,
+            },
+        }
+    },
+    validation_operators={
+        "action_list_operator": {
+            "class_name": "ActionListValidationOperator",
+            "action_list": [
+                {
+                    "name": "store_validation_result",
+                    "action": {"class_name": "StoreValidationResultAction"},
                 },
-                "site_index_builder": {
-                    "class_name": "DefaultSiteIndexBuilder",
-                    "show_cta_footer": True,
+                {
+                    "name": "store_evaluation_params",
+                    "action": {"class_name": "StoreEvaluationParametersAction"},
                 },
-            }
-        },
-        validation_operators={
-            "action_list_operator": {
-                "class_name": "ActionListValidationOperator",
-                "action_list": [
-                    {
-                        "name": "store_validation_result",
-                        "action": {"class_name": "StoreValidationResultAction"},
-                    },
-                    {
-                        "name": "store_evaluation_params",
-                        "action": {"class_name": "StoreEvaluationParametersAction"},
-                    },
-                    {
-                        "name": "update_data_docs",
-                        "action": {"class_name": "UpdateDataDocsAction"},
-                    },
-                ],
-            }
-        },
-        anonymous_usage_statistics={
-          "enabled": True
+                {
+                    "name": "update_data_docs",
+                    "action": {"class_name": "UpdateDataDocsAction"},
+                },
+            ],
         }
-    )
-    ```
+    },
+    anonymous_usage_statistics={
+      "enabled": True
+    }
+)
+```
 
 
 2. **Pass this DataContextConfig as a project_config to BaseDataContext**
```

Comment:
```suggestion
### 2. Pass this DataContextConfig as a project_config to BaseDataContext
```

Reply:
Rather than have these denoted as bold text, it is more appropriate to have them marked as H3 sub sections.  The H1 sub section "Steps" above should be denoted as an H2 sub section (H1 is reserved for the page title).